
---
int8 FAILED = 0
int8 SUCCESS = 1
int8 SKIPPED = 2
int8 status